weatbag/action.py

```diff
@@ -10,24 +11,28 @@ def get_action():
     """
     action = []
     while len(action) == 0:
-        action = input('> ').lower().split()
+        action = input('\n> ').lower().split()
         action = [w for w in action if w not in words.prepositions]
     return action
 
 move_directions = {'n','e','s','w','north','east','south','west'}
-
+mother=["You ought to be ashamed!","Such language for such a bold adventurer!","What would your mother say?"]
 def is_move(do):
-    return len(do) == 2 and (do[0] in words.move) and (do[1] in move_directions)      
+    return len(do) == 1 and (do[0] in words.move)
```

Comment:
Why not use move_directions instead of redefining `words.move`?

I'd also like it if the two-word commands (like 'go east') keep working as well as the short versions.